CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
 
+This release requires all processes accessing a Realm file concurrently to be on
+this version or later. This includes iOS extensions and the Realm Browser.
```

Comment:
I believe older versions will fail to open files written by this version even if it's non-concurrent.